packages/commands/src/limeobject/save-object.error.ts

```diff
@@ -0,0 +1,7 @@
+import { LimeObject } from '@limetech/lime-web-components-interfaces';
+
+export class SaveLimeObjectError extends Error {
+    constructor(public limeobject: LimeObject) {
+        super();
```

Comment:
I think we might want to set a message here as well, e.g.
```suggestion
        super(`Could not save limeobject ${limetype}:${id}`);
```

Reply:
Also, there might be some problems with having a class defined here and it might be better to just have an interface and implement it somewhere else. Having the class here will include it in every addon using `lime-web-components`, so there will be multiple instances of the `SaveLimeObjectError` class. We will throw one version of it from our implementation and the addon might catch it like this, but it will fail since the two classes are not the same even though they have the same name :disappointed: 

```ts
try {
    // save limeobject
} catch (e) {
    if (e instanceof SaveLimeObjectError) {
        // do something special
    }
}

```

Reply:
Ah, good input! 👍

Reply:
Do you think we should keep it here in `commands` though, or should I move it to `interfaces`?

Reply:
Hmm :thinking: Maybe `interfaces` makes more sense?
They will all be merged soon hopefully anyway :sweat_smile:

Reply:
⚡ Moved to `interfaces`.